ddl/ddl_api.go

```diff
@@ -1908,7 +1908,7 @@ func setTemporaryType(ctx sessionctx.Context, tbInfo *model.TableInfo, s *ast.Cr
 	case ast.TemporaryGlobal:
 		tbInfo.TempTableType = model.TempTableGlobal
 		if !ctx.GetSessionVars().EnableGlobalTemporaryTable {
-			return errors.New("global temporary table is experimental and it is switched off by tidb_enable_global_temporary_table")
+			return errors.New("It is switched off by tidb_enable_global_temporary_table")
```

Comment:
I think we can even remove the sysvar unless there is a special use case for disabling it for regression testing? It is not like a special performance optimization, users can easily avoid temporary tables by not using them. Let's see what @djshow832 thinks on this point.

If you do keep the sysvar, because this uses `errors.New`, it won't have a unique SQL error code. So it won't uniquely show up in [client errors](https://docs.pingcap.com/tidb/stable/client-errors-summary-global#client_errors_summary_global) if someone is looking for reason their code isn't working:

```suggestion
			return errors.New("SET tidb_enable_global_temporary_table=1 to enable temporary tables")
```